xyz

```diff
@@ -65,6 +68,7 @@ dir="$(cd -P "$(dirname "$path")" && pwd)"
 branch=master
 increment=patch
 message_template='Version X.Y.Z'
+commit_flags=''
```

Comment:
I'm not sure if a `commit_flags` string over an `edit_message` boolean is a good idea, but I'm no hero in bash, and the boolean made the `git commit` command look pretty messy.